anju.js

```diff
@@ -0,0 +1,21 @@
+const quoteOfTheDay = document.querySelector("#quote-of-the-day");
+  fetch("https://favqs.com/api/qotd")
+    .then((res) => {
+      if (!res.ok) {
+        throw Error("Something ain't right!");
+      }
+      return res.json();
+    })
+    .then((res) => {
+      const quoteBody = res.quote.body
+      const quoteAuthor = res.quote.author
+        const li = document.createElement("li");
+        li.classList.add("quote-li")
+        li.textContent = `"${quoteBody}" -${quoteAuthor}`;
+        quoteOfTheDay.appendChild(li);
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
Maybe you want to do something different besides log the err to the console. Like inform the user to try again later.